cogs/tags.py

```diff
@@ -0,0 +1,22 @@
+import aiohttp
+
+from nextcord.ext import commands
+
+class Tags(commands.Cog):
+    @commands.command()
+    async def tag(self, ctx, *, name: str):
+        """Grabs the Markdown file for a tag."""
+        branch_name = "master" # TODO: Dynamically grab current branch name via git
+        markdown_link = f"https://raw.githubusercontent.com/nextcord/previous/{branch_name}/assets/{name}.md"
+
+        async with aiohttp.ClientSession() as sess:
```

Comment:
This should use the global session defined in main.py
https://github.com/nextcord/previous/blob/master/main.py#L92-L93

Reply:
Oh I didn't realize that existed. Mb